### ISSUE TITLE
can i use trape without google api key?

### ISSUE THREAD

**Issue by cybergost** [jofpin/trape#145]

**Comment by DJCHICKEN4** [comment]
@cybergost Yes, however you will need to enter something or it will produce an error you can enter random letters or numbers for the google api key. Also just a little side note there is no advantage at this time to use a valid google api key because the google maps feature is currently not working.
